src/providers/Youtube/index.js

```diff
@@ -0,0 +1,2 @@
+import {default} from '../../components/Container/index';
```

Comment:
Why are you importing something isn't used?

Reply:
removed